pipelines/deploy-release/README.md

```diff
@@ -9,14 +9,16 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | release | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution | No | - |
 | releaseplan | The namespaced name (namespace/name) of the releasePlan | No | - |
 | releaseplanadmission | The namespaced name (namespace/name) of the releasePlanAdmission | No | - |
-| releasestrategy | The namespaced name (namespace/name) of the releaseStrategy | No | - |
 | snapshot | The namespaced name (namespace/name) of the snapshot | No | - |
 | enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
 | enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
 | verify_ec_task_git_url | The git repo url of the verify-enterprise-contract task | No | - |
 | verify_ec_task_git_revision | The git repo revision the verify-enterprise-contract task | No | - |
 | verify_ec_task_git_pathInRepo | The location of the verify-enterprise-contract task in its repo | No | - |
 
+## Changes since 0.9.0
```

Comment:
This doesn't seem backwards compatible. Per https://github.com/redhat-appstudio/release-service-catalog/pull/212#discussion_r1336968507 shouldn't this be 1.0.0?

Reply:
Wondering the same. Removing a parameter shouldn't break any pipeline passing it, isn't it? But the results... maybe.

Reply:
Yeah, I think Johnny is right. It does seem a bit weird, given what we've been doing until now. But if we want to use semver going forward, we should probably increase the major version here.